documentation/CAMARA-Security-Interoperability.md

```diff
@@ -120,13 +120,16 @@ Scope values determine the specific CAMARA services being requested by the Servi
 
 ### Purpose as part of scope
 
-To improve interoperablitiy this document RECOMMENDS that API Working groups define scope values following this scheme:
+To improve interoperablitiy this document RECOMMENDS that API Working groups define each single scope value following this scheme:
 
 `dpv:<dpvValue>#<technical_scope>`
 
 - `<dpvValue>` is coming from [W3C DPV purpose definition](https://w3c.github.io/dpv/dpv/#vocab-purpose)
 - `<technical_scope>` some technical scope needed by the Camara API
 
+Example value of scope parameter in a request:
+> "openid dpv:<dpvValue>#camara-api-scope0 dpv:<dpvValue>#camara-api-scope-1 dpv:<dpvValue>#camara-api-scope-2"
```

Comment:
See https://github.com/camaraproject/IdentityAndConsentManagement/pull/121/commits/a7156c0a270e152d696af2c17b3944d9178a54a5#r1492745957.

This will be a source of problems without further clarifications that need to be agreed upon. For example, all the `dpv:<dpvValues>` must be identical because that represents the reason why you are processing some personal information (e.g., you may access the sim swap API either for dpv:FraudPrevention#sim-swap or for dpv:Marketing#sim-swap but not both at the same time)

I'll revert this commit until that part is discussed.

cc @Elisabeth-Ericsson I think this is also what you said.